specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2021-07-01/machineLearningServices.json

```diff
@@ -3161,6 +3204,9 @@
           "items": {
             "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
           },
+          "x-ms-identifiers": [
+            "error/code"
```

Comment:
'error/code' or '/error/code'?

Reply:
Should be /error/code. I have updated it in the new iteration.